webrtc.html

```diff
@@ -4830,11 +4842,39 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         </section>
       </div>
       <div>
+        <pre class="idl">enum RTCVadStatus {
+        "disable",
+        "enableIfNegotiated"
+        };</pre>
```

Comment:
you use "disable" and "enable" here, but "disabled" and "enabled" in the table below (the latter sounds better)